src/data/ground/ground_data_a01p01_station.h

```diff
@@ -0,0 +1,1488 @@
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(2),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(6),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(6),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(6),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(6),
+    { 0x54, 0x00,  0x0017,  0x00000000,  0x00000000, NULL },
+    { 0xdd, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(6),
+    END_DELETE,
+};
+
+static const struct ScriptCommand s_gs162_g5_s0_station_sref_script[] = { /* 0x821161c */
+    DEBUGINFO,
+    SELECT_MAP(162),
+    SELECT_ENTITIES(-1, 0),
+    BGM_SWITCH(5),
+    { 0x27, 0x00,  0x0005,  0x00000020,  0x00ffffff, NULL },
+    { 0xe3, 0x00,  0x0003,  0x00000000,  0x00000000, NULL },
+    FANFARE_PLAY2(506),
+    { 0x28, 0x01,  0x0005,  0x00000020,  0x00ffffff, NULL },
+    RET,
+};
+
+static const struct ScriptRef s_gs162_g5_s0_station_sref = { 400, 7, NULL /* EVENT_CONTROL */, s_gs162_g5_s0_station_sref_script }; /* 0x82116ac */
+
+static const struct ScriptCommand s_gs162_g5_s0_lives0_dlg0[] = { /* 0x82116b8 */
+    DEBUGINFO,
+    { 0x53, 0x00,  0x0000,  0x01000000,  0x00000000, NULL },
+    { 0x54, 0x00,  0x0002,  0x00000000,  0x00000000, NULL },
+    { 0xdf, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(60),
+    { 0x91, 0x0a,  0x0001,  0x00000000,  0x00000000, NULL },
+    WAIT(30),
+    { 0x34, 0x00, -0x0001,  0x00000000,  0x00000000, _(" I've been waiting for you.") },
+    { 0x30, 0x00,  0x0000,  0x00000000,  0x00000000, NULL },
+    WAIT(10),
+    { 0x2e, 0x02,  0x0000,  0x0000000c,  0x00000000, NULL },
+    { 0x33, 0x00,  0x0000,  0x00000000,  0x00000000, _("Ga...#W$n2!") },
```

Comment:
`$n2` is a placeholder for a nickname, which is represented by the macro `ARG_NICKNAME_2`. Is it possible to use these argument macros throughout the scripts?